docs/Intro/dashboard.md

```diff
@@ -24,7 +24,13 @@ Using the dashboard, users can select a time interval on the graph and, for each
     width="500"
 />
 </div>
-In addition, users can cryptographically verify in the browser that the validator's signature has not been tampered with.
+
+## Verifiability on Every Level
+
+Users can cryptographically verify in the browser using the Dashboard any **Validator's Signature**, as well as the  the **Schnorr Signature** used for pushing data onchain for the corresponding Oracle.
```

Comment:
The end of the sentence sounds a bit weird? 

Maybe something like: "[...] as well as ~the~ the Schnorr Signature used to push the price onchain to the corresponding Oracle".